Gopkg.lock

```diff
@@ -37,8 +38,7 @@
     "io",
   ]
   pruneopts = "NUT"
-  revision = "9f30cefb0ea118d42f84c30964bbb3a842f004be"
-  version = "v4.5.2"
```

Comment:
I think you didn't run `dep ensure -update github.com/Scalingo/go-scalingo` as this line should be updated to `v4.5.3`